recipes/mathter/all/conanfile.py

```diff
@@ -0,0 +1,45 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class MathterConan(ConanFile):
+    name = "mathter"
+    license = "MIT"
+    homepage = "https://github.com/petiaccja/Mathter"
+    url = "https://github.com/conan-io/conan-center-index/"
+    description = "Powerful 3D math and small-matrix linear algebra library for games and science."
+    topics = ("game-dev", "linear-algebra", "vector-math", "matrix-library")
+    no_copy_source = True
+    settings = "compiler"
+        
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "apple-clang": 10,
+            "clang": 6,
+            "gcc": 7,
+            "Visual Studio": 16,
```

Comment:
Now you can refine a little bit if you have a good knowledge of C++17 features used in your library.

Reply:
Unfortunately, I don't know the specifics. I would have to include older compiler versions in the CI build matrix. I think, for now, as long as the compiler supports -std=c++17, it should work, but I'll try to refine it to make sure.